mentors/javascript/a.json

```
@@ -38,5 +38,13 @@
     "link_url": "https://github.com/alexZielonko",
     "avatar_url": "https://avatars1.githubusercontent.com/u/5274697?s=460&v=4",
     "bio": "I'm one part professional developer and two parts programming enthusiast, shaken with a liberal dash of pragmatism."
+  },
+  {
+    "github_username": "aibrahim3546",
+    "name": "Ahmad Ibrahim",
+    "link_text": "https://aibrahim3546.github.io/",
+    "link_url": "https://aibrahim3546.github.io/",
+    "avatar_url": null,
+    "bio": "I Javascript developer, ReactJs, NodeJs, React Native."
```

```suggestion
    "bio": "I am a Javascript developer, ReactJs, NodeJs, React Native."
```

You may want to clean up your "bio" sentence some more, so it makes more sense.  I am not sure if you are just a Javascript developer and also a developer that works with ReactJS, NodeJS, and React Native?  Let me know if you would like some help with the wording.

---

I have made some adjustment. Have a look.
Thank you for the suggestion. :)